src/components/AuthorCard.jsx

```diff
@@ -0,0 +1,26 @@
+import React from 'react'
+import Link from 'next/link'
+export default function AuthorCard({ author }) {
+  return (
+    <div className="bg-blue-100 dark:bg-gray-800 shadow-md rounded-lg overflow-hidden transition-transform duration-300 hover:scale-105 hover:shadow-xl">
+        <img src={author.profile_image_uri} alt={author.name} className="w-full h-48 object-contain" 
```

Comment:
We mostly use camelCase

Reply:
I have added the convention I am following in the pr description. It's basically camelCase for everything and snake_case for only database and graphql schemas.